libs/hdf-converters/src/xccdf-results-mapper.ts

```diff
@@ -278,142 +299,149 @@ export class XCCDFResultsMapper extends BaseConverter {
               rule_result: {
-                path: '$.Benchmark.TestResult',
-                transformer: (testResult: Record<string, unknown>): unknown =>
-                  getRuleResultItem(testResult, RULE_RESULT_PATHS)
+                path: ['ruleResult']
               },
               value: {
-                path: '$.Benchmark.Value',
-                transformer: (values: Record<string, unknown>[]): unknown => {
-                  return _.find(values, (value: Record<string, unknown>) => {
-                    const id = _.get(value, 'id');
-                    return id && id === valueIdTracker;
-                  });
-                }
+                path: ['values']
               }
             },
             refs: [],
             source_location: {},
-            title: {path: ['Rule.title.text', 'Rule.title']},
-            id: {
-              path: 'Rule',
-              transformer: (input: Record<string, unknown>): string => {
-                const valueIdPaths = ['check.check-export.value-id'];
-                let setValueIdTracker = false;
-                for (const path of valueIdPaths) {
-                  const valueId = _.get(input, path);
-                  if (valueId !== undefined) {
-                    valueIdTracker = valueId as string; // NOTE: global variable
-                    setValueIdTracker = true;
-                  }
-                }
-                if (!setValueIdTracker) {
-                  valueIdTracker = undefined;
-                }
-                const id = _.get(input, 'id');
-                if (typeof id === 'string') {
-                  idTracker = id; // NOTE: global variable
-                  return id.split('_S')[1].split('r')[0];
-                } else {
-                  return '';
-                }
-              }
-            },
+            title: {path: ['title.text', 'title']},
+            id: {path: ['id']},
             desc: {
-              path: ['Rule.description.text', 'Rule.description'],
-              transformer: (description: string): string => {
-                const descTextJson = convertEncodedXmlIntoJson(description);
-                return _.get(descTextJson, 'VulnDiscussion', '') as string;
-              }
+              path: ['description.text', 'description'],
+              transformer: (description: string): string =>
+                _.get(
+                  parseXml(description),
+                  'ProfileDescription',
+                  description
+                ) as string
             },
             descriptions: [
               {
                 data: {
-                  path: 'Rule.check.check-content-ref.name',
+                  path: ['check.check-content-ref.name'],
                   transformer: parseHtml
                 },
                 label: 'check'
               },
               {
                 data: {
-                  path: 'Rule.fixtext.text',
+                  path: ['fixtext', 'fix'],
```

Comment:
![image](https://github.com/mitre/heimdall2/assets/2015821/b1d64e02-83a2-4fd7-82cb-a6fd067027d7)
fix text not being found